### ISSUE TITLE
Cloud application error ?

### ISSUE THREAD

**Issue by FatFritz2** [Torom/BotLi#91]
```
May 26 12:42:11 PM  Traceback (most recent call last):
May 26 12:42:11 PM    File "user_interface.py", line 7, in <module>
May 26 12:42:11 PM      from api import API
May 26 12:42:11 PM    File "/opt/render/project/src/api.py", line 9, in <module>
May 26 12:42:11 PM      from botli_dataclasses import API_Challenge_Reponse, Challenge_Request
May 26 12:42:11 PM    File "/opt/render/project/src/botli_dataclasses.py", line 7, in <module>
May 26 12:42:11 PM      @dataclass
May 26 12:42:11 PM    File "/opt/render/project/src/botli_dataclasses.py", line 9, in API_Challenge_Reponse
May 26 12:42:11 PM      challenge_id: Challenge_ID | None = None
May 26 12:42:11 PM  TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

I tried to start BotLi but it ended up with this error. Its not from my fault i guess the dataclasses.py file has some error  ( used the --matchmaking flag already)

**Comment by Torom** [comment]
I would guess you're not using Python >= 3.10.

**Comment by FatFritz2** [comment]
The cloud already supported python 3.10
And I am wondering why would you guess so ?

**Comment by FatFritz2** [comment]
Thanks, I fixed the issue. The BOT is running great now

**Comment by Torom** [comment]
You fixed the issue by using Python >= 3.10?
